p4-16/psa/PSA.mdk

```diff
@@ -2158,7 +2158,50 @@ The control plane should provide a
 
 ```
 
-# Appendix: Implementation of the `InternetChecksum` extern { #appendix-internetchecksum-implementation }
+# Appendix: Open Issues {@h1:"A"}
+
+As with any work in progress, we have a number of open issues that are under discussion in the
+working group. These are summarized here:
+
+## Action Selectors
+
+The size parameter in the action_selector instance that defines the maximum number of members in a
+selector. In some cases it might be useful to allow the controller to dynamically provision
+resources on the selector or to utilize different selector sizes on different targets, while using
+a common P4 program.
+
+We also need to formalize the interaction of action profiles and action selectors with counters and
+meters.
```

Comment:
might want to qualify by saying 'action selectors with direct counters and meters'.  I don't think there are any open questions about interactions between action selectors and indexed counters and meters.